File: evaluation.m
function evaluation(results)

    disp("Model Used:");
    disp(results.model);

    disp("RMSE:");
    disp(results.rmse);

    % Plot
    figure;
    plot(results.y_test, 'b');
    hold on;
    plot(results.y_pred, 'r');
    legend('Actual','Predicted');

    title(['Model: ', results.model]);
    xlabel('Time Step');
    ylabel('RH');

    saveas(gcf, 'results_plot.png');

end

File: evaluation.py
import matplotlib.pyplot as plt

def evaluation(results):

    print("Model:", results["model"])
    print("RMSE:", results["rmse"])

    plt.figure()
    plt.plot(results["y_test"].values, label="Actual")
    plt.plot(results["y_pred"], label="Predicted")
    plt.legend()
    plt.title(f"Model: {results['model']}")
    plt.xlabel("Time Step")
    plt.ylabel("RH")

    plt.savefig("results_plot.png")
    plt.show()
